modules/configuring-localnet-switched-topology.adoc

```diff
@@ -87,18 +105,17 @@ spec:
 <5> The name for the additional network from which traffic is forwarded to the OVS bridge. This additional network must match the name of the `spec.config.name` field of the `NetworkAttachmentDefinition` CRD that defines the OVN-Kubernetes additional network.
 <6> The name of the OVS bridge on the node. This value is required only if you specify `state: present`.
 <7> The state for the mapping. Must be either `present` to add the bridge or `absent` to remove the bridge. The default value is `present`.
-
-This declarative approach is recommended because the NMState Operator applies additional network configuration to all nodes specified by the node selector automatically and transparently.
-
-The following JSON example configures a localnet secondary network:
-
++
+The following JSON example configures a localnet, `localnet2`, secondary network:
++
 [source,json]
 ----
 {
   "cniVersion": "0.3.1",
   "name": "ns1-localnet-network",
   "type": "ovn-k8s-cni-overlay",
-  "topology":"localnet",
+  "topology":"localnet2",
```

Comment:
Here it should say: `topology: localnet`

Reply:
:heavy_check_mark: